package.json

```diff
@@ -14,25 +14,25 @@
   },
   "dependencies": {
     "@babel/compat-data": "^7.11.0",
-    "@bcrs-shared-components/action-chip": "1.0.27",
-    "@bcrs-shared-components/breadcrumb": "2.0.5",
-    "@bcrs-shared-components/business-lookup": "1.0.3",
-    "@bcrs-shared-components/certify": "2.0.8",
-    "@bcrs-shared-components/completing-party": "2.0.7",
+    "@bcrs-shared-components/action-chip": "1.0.28",
+    "@bcrs-shared-components/breadcrumb": "2.0.6",
+    "@bcrs-shared-components/business-lookup": "1.0.4",
+    "@bcrs-shared-components/certify": "2.0.9",
+    "@bcrs-shared-components/completing-party": "2.0.8",
     "@bcrs-shared-components/confirm-dialog": "1.1.1",
-    "@bcrs-shared-components/contact-info": "1.1.9",
+    "@bcrs-shared-components/contact-info": "1.1.10",
     "@bcrs-shared-components/corp-type-module": "1.0.7",
-    "@bcrs-shared-components/court-order-poa": "2.0.8",
-    "@bcrs-shared-components/date-picker": "1.1.19",
+    "@bcrs-shared-components/court-order-poa": "2.0.9",
+    "@bcrs-shared-components/date-picker": "1.1.20",
     "@bcrs-shared-components/detail-comment": "1.0.13",
-    "@bcrs-shared-components/enums": "1.0.20",
-    "@bcrs-shared-components/fee-summary": "1.1.22",
+    "@bcrs-shared-components/enums": "1.0.21",
+    "@bcrs-shared-components/fee-summary": "1.1.23",
     "@bcrs-shared-components/folio-number": "1.0.3",
     "@bcrs-shared-components/help-business-number": "1.0.0",
-    "@bcrs-shared-components/interfaces": "1.0.40",
-    "@bcrs-shared-components/nature-of-business": "1.0.10",
-    "@bcrs-shared-components/share-structure": "2.0.1",
-    "@bcrs-shared-components/staff-payment": "2.0.10",
+    "@bcrs-shared-components/interfaces": "1.0.41",
+    "@bcrs-shared-components/nature-of-business": "1.0.11",
+    "@bcrs-shared-components/share-structure": "2.0.2",
+    "@bcrs-shared-components/staff-payment": "2.0.11",
```

Comment:
^^ These are only increased by a single patch number, which is expected due to the enum update [**in this PR**](https://github.com/bcgov/bcrs-shared-components/pull/148). As this is the only change, no extra testing is required.